ros/src/waypoint_loader/waypoint_loader.py

```diff
@@ -51,7 +51,7 @@ def load_waypoints(self, fname):
                 p.pose.pose.position.z = float(wp['z'])
                 q = self.quaternion_from_yaw(float(wp['yaw']))
                 p.pose.pose.orientation = Quaternion(*q)
-                p.twist.twist.linear.x = float(self.velocity*0.27778)
+                p.twist.twist.linear.x = float(self.velocity)
```

Comment:
??

Reply:
Not related to DBW itself, but I also took care of using velocities in m/s. The original code was wrong, assuming that `self.velocity` is in km/h (which is not, it's original in mph) converting to m/s.

Reply:
ok so lets add comment describing which unit conversion is done or better add method like mph_to_ms

Reply:
Yep I think that's good and clear.
Actually that's already part of the commit, please note the constructor:

    self.velocity = self.mph_to_ms(rospy.get_param('~velocity'))

I created the function `mph_to_ms` below.